documentation/book/proc-configuring-kafka-mirror-maker-whitelist.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-mirror-maker-whitelist.adoc
+
+[id='proc-configuring-kafka-mirror-maker-whitelist-{context}']
+= Configuring the topics whitelist
+
+The topic whitelist, describing which topics has to be mirrored by Kafka Mirror Maker from source to target Kafka cluster, can be configured using the `whitelist` property in `KafkaMirrorMaker.spec`.
```

Comment:
I think you should use _have_ instead of _has_. @ncbaratta @laidan6000 @dmuntima Can you check for grammar, please?

Reply:
Yes, it should be _have_".

Reply:
Done